src/getXPub.c

```diff
@@ -0,0 +1,93 @@
+/**
+ * Copyright (c) Hathor Labs and its affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+#include <stdint.h>
+#include <stdbool.h>
+#include <os.h>
+#include <os_io_seproxyhal.h>
+#include <string.h>
+#include "hathor.h"
+#include "ux.h"
+
+// Define the approval screen. This is where the user will confirm that they
+// want to authorize access from the desktop wallet.
+static const bagl_element_t ui_getXPub_approve[] = {
+    UI_BACKGROUND(),
+
+    // Rejection/approval icons, represented by a cross and a check mark,
+    // respectively.
+    UI_ICON_LEFT(0x00, BAGL_GLYPH_ICON_CROSS),
+    UI_ICON_RIGHT(0x00, BAGL_GLYPH_ICON_CHECK),
+
+    // The two lines of text, which together form a complete sentence:
+    //
+    //    Authorize
+    //    access?
+    //
+    UI_TEXT(0x00, 0, 12, 128, "Authorize"),
+    UI_TEXT(0x00, 0, 26, 128, "access?"),
+};
+
+// This is the button handler for the approval screen
+static unsigned int ui_getXPub_approve_button(unsigned int button_mask, unsigned int button_mask_counter) {
+    // tx is the offset within G_io_apdu_buffer
+    uint16_t tx = 0;
+    cx_ecfp_public_key_t public_key;
+    cx_ecfp_private_key_t private_key;
+    unsigned char chain_code[32];
+    unsigned char hash_160[20];
+    // bip32 path for 44'/280'/0'/0
+    uint32_t path[] = {44 | 0x80000000, 280 | 0x80000000, 0 | 0x80000000, 0};
```

Comment:
Shouldn't we copy as we did [here](https://github.com/HathorNetwork/ledger-app-hathor/pull/4/files#diff-1dba452b06ff9b969864abb8ad184181R17)?

Reply:
It's already been changed in the other PR. I noticed that I was repeating it so I defined a variable with our bip44 path and always use it when deriving addresses.

But in this PR, as it was only this instance, it was "hardcoded".